microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md

```diff
@@ -54,29 +54,25 @@ Learn more about using protocol handlers with your PWA at [Experimental features
 
 When a user selects the **Settings** (**...**) menu in the app's title bar, the _app info_ menu is shown. We've updated this menu and streamlined the experience for the user in the following ways:
 * Moved the app publisher information to the top level and made it the first thing a user sees.
-* Moved the privacy information and controls into a dedicated 2nd-level **Privacy** menu.
-* Moved content-related tools into a dedicated 2nd-level **More tools** menu.
-
-These changes make the **Settings** menu provide a user experience that's more like a desktop app than a browser UI.
-
-#### Updated App Info Menu
 
 :::image type="complex" source="media/app-info.png" alt-text="The new, streamlined app-info menu" lightbox="media/app-info.png":::
```

Comment:
Add leading spaces to indent the images, to be presented as a child of each list item.   
- For list items, in my experiment, I put * in column 1, then text of the list item started ~column 4.
- For each image, I selected all lines of an image and pressed Tab key to add maybe 4 leading spaces, to match the indent of the text of the list item.  
Then I viewed Preview in VS Code (Sh+Ctrl+V) and it looked good there.

Reply:
Ok updated.